config/courses.json

```diff
@@ -8,7 +8,11 @@
     },
     "deploy": true,
     "devtoolsEnabled": true,
-    "inProduction": true
+    "inProduction": true,
+    "name": "spanish-from-english",
+    "source": "English",
+    "target": "Spanish",
+    "description": "Spanish for English speakers"
```

Comment:
this could be automatically generated, right?

Reply:
TBH I did not know that courses.json is auto generated. IIUC, it is [generate.py](https://github.com/LibreLingo/LibreLingo/blob/main/apps/tools/librelingo_tools/generate.py) that creates it, right? 

So, this PR does not make sense, and we need a new one for the script? And just use what's available in [course](https://github.com/LibreLingo/LibreLingo/blob/d46672e5f91fa0b484c8131dae7bc823e96d21cf/apps/tools/librelingo_tools/generate.py#L85)?

Reply:
no, I don't mean courses.json is auto-generated. I mean, that contents of that field could be auto-generated (in some code or library based on this file) rather than being specified manually here.

Actually it is something that should probably be translated even :thinking:

Reply:
The PR makes sense overall

Reply:
I changed my mind and I will approve and merge this. The reason is that the description field could be used as a generic comment field rather than an automatically generated title for the course